src/lib/libfunctional/functional.cc

```diff
@@ -38,8 +38,8 @@ void Functional::common_init()
     gga_ = false;
     meta_ = false;
     name_ = "";
```

Comment:
It'd be pretty hard to build a functional and use it w/o setting names. But out of an abundance of caution, some spaces would probably be good.